kb/sd_algorithms_and_functions/nrel_activation_function.scs

```diff
@@ -23,8 +39,49 @@ nrel_activation_function=>nrel_idtf:[функция возбуждения](* <-
 			-> rrel_example: [Механизм любой нейронной сети подразумевает наличие функции активации](* <- lang_ru;; *);;
 		*);;
 	*);; 
-nrel_activation_function => nrel_first_domain: concept_neuron_node;;
- nrel_activation_function => nrel_second_domain: concept_neuron_node;;
+nrel_activation_function => nrel_first_domain: formal_neuron;;
+nrel_activation_function => nrel_second_domain: function;
+
+
+=> nrel_subdividing: ...
+(*
+    <- sc_node_tuple;;
+    => nrel_main_idtf: [линейная функция] (* <- lang_ru;; *);;
+		
+
+*);
+
+=> nrel_subdividing: ...
+(*
+    <- sc_node_tuple;;
+    => nrel_main_idtf: [пороговая функция] (* <- lang_ru;; *);;
+*);
+
+=> nrel_subdividing: ...
+(*
+    <- sc_node_tuple;;
+    => nrel_main_idtf: [сигмоидная функция] (* <- lang_ru;; *);;
+*);
+
+=> nrel_subdividing: ...
+(*
+    <- sc_node_tuple;;
+    => nrel_main_idtf: [функция гиперболического тангенса] (* <- lang_ru;; *);;
+*);
+
+=> nrel_subdividing: ...
+(*
+    <- sc_node_tuple;;
+    => nrel_main_idtf: [функция ReLU] (* <- lang_ru;; *);;
+*);
+
+=> nrel_subdividing: ...
+(*
+    <- sc_node_tuple;;
+    => nrel_main_idtf: [функция softmax] (* <- lang_ru;; *);;
+*);;
```

Comment:
```suggestion
=> nrel_subdividing: {
    concept_threshold_function;
    concept_sigmoid_function;
    concept_hyperbolic_tangent_function;
    concept_relu_function;
    concept_softmax_function
*);
```

Identificators (ru and en) of these functions classes should be specified in separate file.

Fix subdivinding in while your PR